Moving FAQ  Dataset Metadata Errors  to GTN

Beachyesh opened the pull request:
This file is meant to answer the FAQ:  How do I find, adjust, and/or correct metadata?

Beachyesh commented:
Thanks a lot @beatrizserrano for all the inputs, this clears up a lot of doubts I had while making this file. Will make these changes.

beatrizserrano commented:
> @beatrizserrano Does this file seem more coherent with our goals? Let me know. Thanks!

yes, this looks better now :) thanks!

Beachyesh commented:
> > @beatrizserrano Does this file seem more coherent with our goals? Let me know. Thanks!
> 
> yes, this looks better now :) thanks!

That's great! Any other changes before merging?

beatrizserrano commented:
> > > @beatrizserrano Does this file seem more coherent with our goals? Let me know. Thanks!
> > 
> > 
> > yes, this looks better now :) thanks!
> 
> That's great! Any other changes before merging?

yes, the suggestions I made in my comment above: clear title, clear steps, structured metadata...

Beachyesh commented:
> > > > @beatrizserrano Does this file seem more coherent with our goals? Let me know. Thanks!
> > > 
> > > 
> > > yes, this looks better now :) thanks!
> > 
> > 
> > That's great! Any other changes before merging?
> 
> yes, the suggestions I made in my comment above: clear title, clear steps, structured metadata...

Okay, so in this updated file, I have taken a specific issue of missing dataset metadata and given a small solution in a few short steps. Accordingly, I have given it a suitable title (to my best knowledge). I also think that a separate faq file for "tool errors related to metadata" and "changing or updating metadata" would be helpful. I can work on it once my other PRs pass their reviews.

annefou commented on faqs/galaxy/dataset_metadata_missing.md:
```diff
@@ -0,0 +1,22 @@
+---
+title: Missing or Incomplete Dataset metadata 
+area: datasets   
```
I am not sure the title reflects what an end users would look for. I mean you would need to know that you have an issue with the metadata of your dataset.

annefou commented on faqs/galaxy/dataset_metadata_missing.md:
```diff
@@ -0,0 +1,22 @@
+- Tools error when using a previously successfully used specific dataset
+- Tools error with a message that ends with: ``OSError: [Errno 2] No such file or directory.``
+
+Solution:
+Click on the the dataset's {% icon galaxy-pencil %} ✏️ to reach the _Edit Attributes_ forms and click either:
```
The formatting will not appear very nicely here. You either need to add a list (or numbered list) or have a blank line in between.

annefou commented on faqs/galaxy/dataset_metadata_missing.md:
```diff
@@ -0,0 +1,22 @@
+
+Solution:
+Click on the the dataset's {% icon galaxy-pencil %} ✏️ to reach the _Edit Attributes_ forms and click either:
+- **Directly reset metadata** 
+  - Find the tab for the metadata you want to change, make the change, and save.
+- **Autodetect metadata**
```
So here maybe end users would like to know if they should first reset metadata and then autodetect metadata, or should they choose which one to do? That would be very nice to clarify it. Many thanks.

Beachyesh commented:
@annefou 
That was a very valid point to consider, how the user will know if they have an issue with metadata, which is why I have added a small introduction in the beginning and renamed the file to "tool errors relating to metadata". 
Also added a line after the Solution to make the format better. 
Finally, also mentioned that the user has to choose one of the two options from the solution as it applies to their situation.

This has made the whole faq slightly longer. Please let me know if that is okay. Looking forward to your review. Thanks!

Beachyesh commented on faqs/galaxy/dataset_tool_erorrs_metadata.md:
```diff
@@ -0,0 +1,28 @@
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action
+ - Incorrect autodetection of datatypes, which need manual modification
+ - Undectected attributes, which require user action (example: assigning database to newly uploaded data)
```
Oops! Fixed the typo.

annefou commented on faqs/galaxy/dataset_tool_erorrs_metadata.md:
```diff
@@ -0,0 +1,28 @@
+ - Undetected attributes, which require user action (example: assigning database to newly uploaded data)
+
+How to notice missing Dataset Metadata:
+- Dataset will not be downloaded when using the disk icon.
```
you could use icons here (see https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/faqs/icons_list.html).

Beachyesh replied:
Done.

annefou commented on faqs/galaxy/dataset_tool_erorrs_metadata.md:
```diff
@@ -0,0 +1,29 @@
+title: Tool Errors related to Metadata
+area: datasets   
+description:  Finding and Correcting Metadata
+contributors: jennaj,
```
It looks weird here.

Beachyesh replied:
My mistake.

annefou replied:
it should not be removed but updated to reflect the effective list of current contributors. I guess you want to be in that list too.

Beachyesh replied:
Yes, I'm aware, I was confused as to whose names to be added? Just mine or also someone else's?

beatrizserrano replied:
there's not such thing as mandatory contributors here :) Here's the explanation: jennaj wrote the FAQ initially, so she has to be on the contributors list. You moved it to the GTN, so you also deserve to be there. Just add both names (please check how it's done in the other PRs)

beatrizserrano commented on faqs/galaxy/dataset_tool_erorrs_metadata.md:
```diff
@@ -0,0 +1,29 @@
+---
+title: Tool Errors related to Metadata
```
hmm I thinking now that we may want to have the title as an explicit question, what do you think @Beachyesh?

Beachyesh replied:
Okay, so this one has felt a little tricky. At first, I kept the title about missing Metadata, but Anne pointed out how will the user know that the error is about the missing metadata? I thought it was a fair point and hence I made the decision to keep the FAQ title slightly broader. Considering this suggestion and usual title guidelines in mind does "Incorrect or Undetected Datatype Attributes" sound like an appropriate title?

annefou replied:
Well, we do not know everything and this is why collaborating together is usually best!

Maybe something like: "How to find and correct tool Errors related to Metadata?"

Beachyesh replied:
Yess sounds good!

beatrizserrano commented on faqs/galaxy/dataset_tool_erorrs_metadata.md:
```diff
@@ -0,0 +1,29 @@
+title: Tool Errors related to Metadata
+area: datasets   
+description:  Finding and Correcting Metadata
+contributors: [jennaj, beachyesh]
+layout: faq         
```
same comment as in the other PR with the order here

beatrizserrano commented on faqs/galaxy/dataset_tool_erorrs_metadata.md:
```diff
@@ -0,0 +1,29 @@
+---
+
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action
```
```suggestion
 - Tool outputs, which are automatically assigned without user action.
```

beatrizserrano commented on faqs/galaxy/dataset_tool_erorrs_metadata.md:
```diff
@@ -0,0 +1,29 @@
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action
+ - Incorrect autodetection of datatypes, which need manual modification
+ - Undetected attributes, which require user action (example: assigning database to newly uploaded data)
```
```suggestion
 - Undetected attributes, which require user action (example: assigning database to newly uploaded data).
```

beatrizserrano commented:
@Beachyesh could you rename the file? errors instead of erorrs :)

annefou commented on faqs/galaxy/dataset_tool_errors_metadata.md:
```diff
@@ -0,0 +1,29 @@
+
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action.
+ - Incorrect autodetection of datatypes, which need manual modification
```
There is no "." at the end of this sentence. It is recommended to be consistent.

annefou replied:
And I see @beatrizserrano already mentioned it! 
It is looking great and we'll be able to merge it soon! Thanks.

Beachyesh replied:
Thanks!

beatrizserrano commented:
thank you @Beachyesh!
